index.html

```diff
@@ -166,6 +166,15 @@ <h4 class="label">Settings</h4>
                 <button @click="reScan" class="mj-rescan">
                   Rescan
                 </button>
+              
+              <br>
+              <br>
+              
+                <input placeholder="Username" class="mj-input" type="text">
+                <input type="hidden" v-model="init">
```

Comment:
We don't need this input for now. It was a dummy input specifically for `lookupDir`.

Reply:
If the input isn't there then where will the user enter its username?

Reply:
I was talking about the one on line 174 which has `type` = `hidden`